CHANGELOG.md

```diff
@@ -198,6 +198,7 @@ All notable changes to this project will be documented in this file.
 [`unit_cmp`]: https://github.com/Manishearth/rust-clippy/wiki#unit_cmp
 [`unnecessary_mut_passed`]: https://github.com/Manishearth/rust-clippy/wiki#unnecessary_mut_passed
 [`unneeded_field_pattern`]: https://github.com/Manishearth/rust-clippy/wiki#unneeded_field_pattern
+[`unsafe_removed_from_name`]: https://github.com/Manishearth/rust-clippy/wiki#unsafe_removed_from_name
```

Comment:
On top of the file:
```diff
 ## Unreleased
- * New lint: [`temporary_cstring_as_ptr`]
+ * New lints: [`temporary_cstring_as_ptr`] and [`unsafe_removed_from_name`]

 ## 0.0.63 — 2016-04-08
```

The script only updates the links because I did not feel like parsing markdown to update the right section :sweat_smile:  (the changelog is new, it wasn‘t there last time you made a PR).